wiki/BBCode/en.md

```diff
@@ -1,581 +1,417 @@
+
+- Dedicated button: ![Underline button](img/underline.jpg "Underline button")
+- Notes:
+  - Use this tag sparingly.
+  - Overuse of this will create discomfort for the reader's eyes.
+  - Overuse of this will weaken the emphasize overall.
+- Syntax:
+```
+[u] ... [/u]
+```
+
+### Strike
+
+_Also known as **strikethrough**._
+
+This will cross out an incorrect word(s) or an entire paragraph.
+
+- Dedicated button: ![Strike button](img/strike.jpg "Strike button")
+- Notes:
+  - Use this tag sparingly.
+  - Overuse of this will create discomfort for the reader's eyes.
+- Syntax:
+```
+[strike] ... [/strike]
+```
+
+### Colour
+
+_BBCode calls this "Color"._
+
+This will give colour words or an entire paragraph.
+
+- Dedicated buttons:
+![The colour box](img/colour.jpg "The colour box")
+- Notes:
+  - This is located on the right side of the text box.
+  - Use this tag sparingly.
+  - Overuse of this will create discomfort for the reader's eyes.
+  - Misuse of this will make your text illegible.
+  - If you choose a colour, make sure it is legible so that anyone can read it.
+  - The `HEXCODE` argument does not need quotation marks.
+- Syntax:
+  - Where `HEXCODE` is a colour in hexadecimal or colour name.
+    - If using hexadecimal, it must start with `#`, followed by 6 hexadecimal (0 - 9, A - F) characters.
+    - If using colour name, it must be a vaild html colour.
+```
+[color=HEXCODE] ... [/color]
+```
+
+For a list of colour names, see [X11 color names](https://en.wikipedia.org/wiki/X11_color_names#Color_name_chart).
+
+### Font Size
+
+This will make the text smaller or larger.
+
+- Dedicated buttons:
+![Font size options](img/font-size.jpg "Font size options")
+- Notes:
+  - Use this tag sparingly.
+  - There are only 4 sizes that you can use:
+    - `50` (tiny)
+    - `85` (small)
+    - `100` (normal; default)
+    - `150` (large)
+  - If you enter a value that is not one of the listed above, it will use the default font size.
+  - The `NUMBER` argument does not need quotation marks.
+- Syntax:
+  - Where `NUMBER` is one of the listed sizes (`50`, `85`, `100`, or `150`).
+```
+[size=NUMBER] ... [/size]
+```
+
+### Spoiler
+
+_Not to be confused with [Spoilerbox](#spoilerbox)._
+
+This will hide text with a black background.
+This can be useful if you are talking about a critical scene in a TV show, movie, or anime but do not want to spoil it for others.
+The reader, however, can still read the text by highlighting over the text.
+
+- Dedicated button: ![Spoiler button](img/spoiler.jpg "Spoiler button")
+- Notes:
+  - If you use the [colour](#colour) tag along with this, the colour will be used with a black background.
+- Syntax:
+```
+[spoiler] ... [/spoiler]
+```
+
+### Spoilerbox
+
+_Not to be confused with [Box](#box) or [Spoiler](#spoiler)._
+
+This will hide a paragraph from the reader's view.
+The reader, however, can still read the paragraph by clicking on the spoilerbox to toggle the hidden paragraph.
```

Comment:
no? the behavior of a spoilerbox is the same as a box (click on the "collapsed text" button to reveal the text)... unless something new happened at the forums?